src/main/java/net/dv8tion/jda/api/EmbedBuilder.java

```diff
@@ -386,6 +409,28 @@ public EmbedBuilder setTimestamp(@Nullable TemporalAccessor temporal)
         return this;
     }
 
+    /**
+     * The {@link java.awt.Color Color} of the embed
+     *
+     * @return Color the builder has been set with.
+     */
+    @Nonnull
+    public java.awt.Color getColor()
+    {
+        return new Color(color);
+    }
+
+    /**
+     * The {@link Int int} of the color as a raw integer
+     *
+     * @return Raw color the builder has been set with.
+     */
+    @Nonnull
+    public int getColorRaw()
+    {
+        return color;
+    }
```

Comment:
You shouldn't annotate primitives with nullability annotations

Reply:
Alright will fix, thanks.